### PR TITLE
TCOMP-1787 : instance not null of Component Manager

### PR DESCRIPTION
**Pull request by clesaec**
https://jira.talendforge.org/browse/TCOMP-1787

### Requirements

* Any code change adding any logic **MUST** be tested through a unit test executed with the default build
* Any API addition **MUST** be done with a documentation update if relevant

### CODE REVIEW

File: component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java [hunk 1]
```diff
@@ -211,13 +203,16 @@ public void run() {
 
                 @Override
                 public void close() {
+                    log.info("close component manager.");
```
Comment (undx) [inline component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java:206]: - set message to `Closing ComponentManager`
- set level to debug

File: component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java [hunk 2]
```diff
@@ -211,13 +203,16 @@ public void run() {
...
                     if (!closed.compareAndSet(false, true)) {
+                        log.info("Component manager already closed");
```
Comment (undx) [inline component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java:208]: - set message to `ComponentManager is already closed`
- set level to debug

File: component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java [hunk 3]
```diff
@@ -211,13 +203,16 @@ public void run() {
...
                         return;
                     }
                     try {
                         synchronized (CONTEXTUAL_INSTANCE) {
                             if (CONTEXTUAL_INSTANCE.compareAndSet(this, null)) {
                                 try {
+                                    log.info("Component manager : remove shutdown");
```
Comment (undx) [inline component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java:215]: - set message to `ComponentManager : removing shutdown hook`
- set level to debug

File: component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java (hunk 1 above)
Comment (clesaec) [reply component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java]: done

File: component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java (hunk 2 above)
Comment (clesaec) [reply component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java]: done

File: component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java (hunk 3 above)
Comment (clesaec) [reply component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java]: done

**Review by undx — APPROVED** [review]
LGTM

**Pull request merged**